components/dashboard/SentimentTracker/SentimentComplete.js

```diff
@@ -7,35 +7,62 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
 
+const SENTIMENT_TYPES = {
```

Comment:
Please DRY this up with https://github.com/newjersey/career-network/blob/c507d82746ff2f8d14f9a723f026964105df1758/components/dashboard/SentimentTracker/SentimentTracker.js#L90

Reply:
Add messages to sentiments, and removed the SENTIMENT_TYPES